packages/sitecore-jss-react/src/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component, ErrorInfo, ReactNode } from 'react';
+
+interface ErrorState {
+  error?: Error;
+  errorInfo?: ErrorInfo;
+}
+
+type ErrorComponentProps = {
```

Comment:
Duplicate of `ErrorComponentProps` defined in `PlaceholderCommon.tsx`. We should consolidate/re-use. 

And looking at this, shouldn't we be able to be more specific with this type? i.e.

```
export type ErrorComponentProps = {
  error?: Error;
};
```